Makefile

```diff
@@ -24,7 +24,8 @@ IMAGE_REGISTRY :=registry.svc.ci.openshift.org
 # $2 - image ref
 # $3 - Dockerfile path
 # $4 - context directory for image build
-$(call build-image,ocp-cluster-kube-controller-manager-operator,$(IMAGE_REGISTRY)/ocp/4.2:cluster-kube-controller-manager-operator, ./Dockerfile.rhel7,.)
+IMAGE_TAG?=$(IMAGE_REGISTRY)/ocp/4.2:cluster-kube-controller-manager-operator
+$(call build-image,ocp-cluster-kube-controller-manager-operator,$(IMAGE_TAG), ./Dockerfile.rhel7,.)
```

Comment:
@sallyom please fix that in a followup.

Reply:
yes, i saw that, mentioned in slack @deads2k can fix here, or whatever